src/Controller/HomeController.php

```diff
@@ -2,27 +2,58 @@
 
 namespace App\Controller;
 
-use App\Contributors\Service\Repository as ContributorsRepository;
-use App\Contributors\Model\Contributor;
-use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use App\Service\PoserInterface;
+use App\Service\ReaderInterface;
+use PUGX\Poser\Poser;
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\HttpFoundation\Response;
 
 class HomeController extends Controller
 {
+
+    /**
+     * @var ReaderInterface
+     */
+    private $statsReader;
+
     /**
-     * @Route("/", name="homepage")
-     * @param ContributorsRepository $contributorsRepository
-     * @return Response
+     * @var Poser
      */
-    public function index(ContributorsRepository $contributorsRepository): Response
+    private $poser;
+
+    /**
+     * HomeController constructor.
+     * @param ReaderInterface $statsReader
+     * @param Poser $poser
+     */
+    public function __construct(ReaderInterface $statsReader, Poser $poser)
+    {
+        $this->statsReader = $statsReader;
+        $this->poser = $poser;
+    }
```

Comment:
if you use these services only in `indexAction` would not it be better to inject them only to the `indexAction`?